app/build.gradle.kts

```diff
@@ -86,6 +99,8 @@ dependencies {
     androidTestImplementation(libs.androidx.espresso.core)
     androidTestImplementation(platform(libs.androidx.compose.bom))
     androidTestImplementation(libs.androidx.ui.test.junit4)
+    androidTestImplementation(libs.androidx.navigation.testing)
+    androidTestImplementation(libs.mockk.android)
```

Comment:
Note that mockk is already an android test dependency in main.

Reply:
Solved with the rebase